src/erlcloud_ec2.erl

```diff
@@ -2415,6 +2420,178 @@ request_spot_instances(Request, Config) ->
             {error, Reason}
     end.
 
+
+decode_spot_fleet_state("submitted") -> submitted;
```

Comment:
is there really need this `list_to_atom()` conversion?
could you make it inline with the rest of the module and just leave it as `{atom(), list()}`